src/Nav.php

```diff
@@ -304,9 +304,10 @@ private function renderItem(array $item): string
 
         if ($this->dropdown) {
             return
-                html::beginTag('div', $options) . "\n" .
+                Html::beginTag('div', $options) . "\n" .
                     Html::a($label, $url, ['class' => 'navbar-link']) . "\n" .
-                    $items;
+                    $items. "\n" .
+                Html::endTag('div');
```

Comment:
It's in a bit of a weird place, but that's already done [here](https://github.com/yiisoft/yii-bulma/blob/master/src/Nav.php#L30)

Reply:
This tag closes submenu in item.

Reply:
> 
> 
> It's in a bit of a weird place, but that's already done [here](https://github.com/yiisoft/yii-bulma/blob/master/src/Nav.php#L30)

Really, a weird place)) remove